library/std/src/io/stdio.rs

```diff
@@ -437,6 +437,28 @@ impl Read for Stdin {
     }
 }
 
+#[stable(feature = "rust1", since = "1.0.0")]
```

Comment:
Please choose some other unique feature name and set the stabilization version to "CURRENT_RUSTC_VERSION".